core/src/services/azdls/backend.rs

```diff
@@ -133,6 +133,15 @@ impl AzdlsBuilder {
         self
     }
 
+    /// Set sas_token of this backend.
+    pub fn sas_token(mut self, sas_token: &str) -> Self {
```

Comment:
Reqsign has `sas_token` support; we can use it here: https://docs.rs/reqsign/0.16.0/reqsign/struct.AzureStorageConfig.html#structfield.sas_token

Maybe it's worth a separate PR?

Reply:
Yes, we can construct like `s3` does:

https://github.com/apache/opendal/blob/3fe8caa08a7e91fe05d210154b792183bd846421/core/src/services/s3/backend.rs#L745-L783